src/integrations/admin/addon-installation/dialog-integration.php

```diff
@@ -51,6 +52,7 @@ public function __construct( \WPSEO_Addon_Manager $addon_manager ) {
 	 */
 	public function register_hooks() {
 		add_action( 'admin_init', [ $this, 'start_addon_installation' ] );
+		add_action( 'admin_init', [ $this, 'check_addons_installed' ] );
```

Comment:
Have you considered making a separate Integration for this logic?
I'd prefer to split this up, as we've already placed the installation logic in a separate folder.

Reply:
I hadn't, mostly because i was expecting it to be very small. certainly can do though.